src/components/InMoreLanguageSwitch.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<div class="wikibase-termbox-subsection-switch">
+		<div class="wikibase-termbox-subsection-switch__switch">
+			<a :href="link" @click="clientShowAllLanguages( $event )">{{ message( MESSAGE_KEYS.ALL_LANG ) }}</a>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import { VueConstructor } from 'vue';
+import Component, { mixins } from 'vue-class-component';
+import Messages, { MessagesMixin } from './mixins/Messages';
+
+@Component
+export default class TermBox extends ( mixins( Messages ) as VueConstructor<MessagesMixin> ) {
+	public readonly link: string;
```

Comment:
This should be solvable through component annotation
```
@Component( {
	link: {
		href: String,
		default: 'place-holder-link',
	},
} )
```

Reply:
Well. To me it feels like passing the buck of testing that prop to the next guy who has to make the link injectable - as such, though generally agreeing with what you said, I feel this does not apply here.

Reply:
From my point of view, I prefer to get the link from the store (because of that single source of truth thing)...I guess there will be a something in between of both approaches...but first of all we need the link, then we can discuss to tackle the problem, right? So it's resolved?